specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/virtualWan.json

```diff
@@ -6047,6 +6047,10 @@
         "preferredRoutingGateway": {
           "$ref": "#/definitions/PreferredRoutingGateway",
           "description": "The preferred gateway to route on-prem traffic"
+        },
+        "hubRoutingPreference": {
+          "type": "string",
```

Comment:
Should we use another "$ref": "#/definitions/HubRoutingPreference", and add Enum in it

Reply:
Updated as per your suggestion